javascript/app.js

```diff
@@ -500,14 +521,15 @@ FinancialFreedom.controller('TimeToRetirementController', function($scope, Retir
         $scope.retirement.already_retired_shown = retirement_data.can_retire_immediately;
         CreateRetirementGraphService.createRetirementGraph(retirement_data);
     };
-    
+
     $(window).resize(function() {
-        CreateRetirementGraphService.createRetirementGraph(retirement_data);
+        var retirement_data = RetirementCalculatorService.calculateRetirementInfo();
+        $scope.refreshOutput(retirement_data);

```

Comment:
A useful improvement at some point in the future would be to just redraw the graph instead of re-calculating everything on a window resize. The data shouldn't change, just the drawing of the graph. I think I initially put this in here because I was being lazy and it was easier at the time to re-do everything than split up the drawing and the calculation logic.